### PR TITLE
Feat/navigation menu

### LINKED ISSUE #24: main navigation
when user hit the logo btn, opens main nav.
need to handle user is not logged, user is logged in.

### PR DESCRIPTION
**Pull request by Ian-Bright**
Implement navigation menu to resolve: https://github.com/State-Channel-4/frontend/issues/24

Note: Sign-in logic currently hardcoded until web3auth is implemented

### CODE REVIEW

File: components/main-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import Image from "next/image"
+import Link from "next/link"
+import browserIcon from "@/assets/browser-icon.svg"
+import xIcon from "@/assets/x-icon.svg"
+
+interface MainMenuProps {
+  onClose: () => void
+}
+
```
Comment (thebeyondr) [inline components/main-menu.tsx:9]: available in site config

**Review by thebeyondr — CHANGES_REQUESTED** [review]

File: components/main-menu.tsx (hunk 1 above)
Comment (Ian-Bright) [reply components/main-menu.tsx:12]: Fixed in: https://github.com/State-Channel-4/frontend/pull/27/commits/0e40b098777b2db2cbadddf0370f87d6795702c9

**Comment by Ian-Bright** [comment]
Implemented the new navigation options:

Screenshot 2023-10-08 at 9 36 17 AM

there isn't a lot of space on mobile. I wonder if we could shrink the text in mobile or maybe add a scroll effect @thebeyondr ?

**Comment by thebeyondr** [comment]
> there isn't a lot of space on mobile. I wonder if we could shrink the text in mobile or maybe add a scroll effect @thebeyondr ?

Both maybe. Lets make the text a bit smaller and if it still overflows, add a scroll. 

Also I think the options should be sentence case instead of title case. (i.e. "Choose tags" instead of "Choose Tags")

**Comment by Ian-Bright** [comment]
> > 
> Both maybe. Lets make the text a bit smaller and if it still overflows, add a scroll.
> 
> Also I think the options should be sentence case instead of title case. (i.e. "Choose tags" instead of "Choose Tags")

Shrunk the text padding, and margin for mobile. Think it looks pretty good.

Also changed to sentence case 🫡

**Pull request merged**